packages/react-components/components/PhoneNumberInput.test.tsx

```diff
@@ -1,7 +1,20 @@
 import PhoneNumberInput from '@celo/react-components/components/PhoneNumberInput'
+import { shallow } from 'enzyme'
 import * as React from 'react'
 import { fireEvent, render } from 'react-native-testing-library'
 
+jest.mock('@celo/react-native-sms-retriever', () => {
+  return {
+    requestPhoneNumber: jest.fn(() => '+49030111111'),
+  }
+})
+
+jest.mock('react-native-device-info', () => {
+  return {
+    getBaseOs: jest.fn(() => 'Android'),
+  }
+})
+
 describe('PhoneNumberInput', () => {
   describe('when defaultCountry is falsy', () => {
     it.skip('renders an AutoComplete and a country can be selected', () => {
```

Comment:
It looks like @cmcewen added a skip to my new test here. Can you please remove it?
I fixed the tests here to work without needing the `__mocks__/country-data.ts`file, which I removed but I guess he added back. Would be nice to remove that mock again and just test with the real data